### PR TITLE
Ck feature1 registration info

### PR DESCRIPTION
**Pull request by fxnggg**
### Feature 1 Registration Information Retrieval Before Submission

Tasks Completed:

- [x] Modify values of tech stack languages
- [x] Add in field for tech stack inputted
- [x] Remove grad checkbox from registration form
- [x] Remove graduate category retrieval before submission
- [x] Create custom pipe to format tech stack data for its retrieval before submission
- [x] Group tech stack and other lang form control fields together for tech stack category retrieval before form submission
- [x] Send boolean value of graduate student in form as well

### CODE REVIEW

File: src/app/features/common-components/registration/registration.component.html [hunk 1]
```diff
@@ -165,6 +165,11 @@ <h4 class="data team-name"> {{ signUpForm.get("teamName").value }} </h4>
             <p class="category">Email</p>
             <p class="data"> {{ signUpForm.get("schoolEmailAddress").value }} </p>
           </div>
+          <div class="info-group">
```
Comment (alison-nicole) [inline src/app/features/common-components/registration/registration.component.html:164]: Is this where the tech stack of a participant is saved? If not, can you tell me how is it stored?

Comment (fxnggg) [reply src/app/features/common-components/registration/registration.component.html:164]: By saved, do you mean where it's stored or where it's submitted? 

When the form is submitted, I believe the sign up service makes a request to the API/backend of the application to post the data from the sign up form. This specific portion of the page is where the user is able to review what they have inputted into the form so far before submitting the form and completing their registration. 

With the tech stack, when the form is submitted what should be sent to the backend is the names of the languages the participant knows from the checkboxes in the form of an array of strings and that's under the form control techStack. Any additional languages/frameworks will be stored in the otherLang form control and that will be just a string. I'm not sure if any of the information I provided answered your questions, so please let me know.

Comment (alison-nicole) [reply src/app/features/common-components/registration/registration.component.html:164]: okay, it is sent in the form of an array of strings. Awesome, that answers my question!

**Review by alison-nicole — APPROVED** [review]
All changes look good. Nice work!

**Review by GurnamanSingh0915 — APPROVED** [review]

**Review by alison-nicole — APPROVED** [review]
Awesome! This fixes the backend problem.

**Pull request merged**